docs/PowerFX/TestEngine.PlaywrightAction.md

```diff
@@ -0,0 +1,36 @@
+# TestEngine.PlaywrightAction
```

Comment:
Move to experimental namespace and add aseembly scanning for the loaded csx to be validating against allowed/denied namespaces